datasets/sample.sql

```diff
@@ -548,7 +548,7 @@ INSERT INTO "Program__c" VALUES(5,'Drug and Alcohol Recovery Management','Recove
 INSERT INTO "Program__c" VALUES(7,'Financial Literacy Program','The Financial Literacy program teaches students the basics of money management: budgeting, saving, debt, investing, and giving. That knowledge lays a foundation for students to build strong money habits early on and avoid many of the mistakes that lead to lifelong money struggles.','','Education','2020-08-03','Active','Teaches students the basics of money management.','Children aged 15-18 enrolled in school district 31');
 INSERT INTO "Program__c" VALUES(8,'Housing Assistance Program','The Housing Assistance program provides assistance for families with children under the age of 18. The family shelters provide both short-term ( and also longer term up to 3 months) emergency shelter. Case management services are available to applicants including referrals to other non-profit needed resources, counseling, food, personal hygiene supplies and free bus tickets. The primary goal of the program is to help families find more permanent housing.','','Housing','2020-03-31','Active','Providing assistance for families with children under the age of 18.','Families with children under the age of 18');
-INSERT INTO "Program__c" VALUES(9,'ESL Program','The primary objective of the ESL program is to teach the students how to read, write, speak and listen to English to be more successful in the all-English classroom setting. Spanish is the language spoken most often by our English learners. Other languages include: German, Chinese, Haitian Creole, Arabic, and some Indian Dialects from Mexico and Guatemala.','2021-04-29','','2020-08-08','Active','A language instruction educational program for students whose primary language is not English.','A practical conversational  English class for those whose primary language is not English.');
+INSERT INTO "Program__c" VALUES(9,'ESL Program','The primary objective of the ESL program is to teach the students how to read, write, speak and listen to English to be more successful in the all-English classroom setting. Spanish is the language spoken most often by our English learners. Other languages include: German, Chinese, Haitian Creole, Arabic, and some Indian Dialects from Mexico and Guatemala.','2022-04-29','','2020-08-08','Active','A language instruction educational program for students whose primary language is not English.','A practical conversational  English class for those whose primary language is not English.');
```

Comment:
Not a big deal, but there is an extra space in this sentence (after the word conversational): A practical conversational  English class for those whose primary language is not English